src/bin/stratisd.rs

```diff
@@ -85,20 +123,45 @@ fn run() -> StratisResult<()> {
                          revents: 0,
                          events: libc::POLLIN,
                      });
-            1
+            2
         }
-        None => 0,
+        None => 1,
     };
 
     // Don't timeout if eventable lets us be event-driven
-    let poll_timeout = if engine_fds_end_idx != 0 { -1 } else { 10000 };
+    let poll_timeout = if engine_fds_end_idx != 1 { -1 } else { 10000 };
 
     loop {
+        // Process any udev block events
+        if fds[0].revents != 0 {
+            loop {
+                match udev.receive_event() {
+                    Some(event) => {
+                        if let Some(added) = handle_udev_add(&event) {
+                            let pool_uuid = engine.borrow_mut().block_evaluate(added.0, added.1);
+                            if let Some(_pool_uuid) = pool_uuid {
+                                #[cfg(feature="dbus_enabled")]
```

Comment:
I am surprised that our tests pass with this as this leaves an empty code block when ```dbus_enabled``` is false. But I guess the compiler must not mind.

Reply:
The compiler did mind until I renamed `pool_uuid` to `_pool_uuid` instead.

Reply:
Huh. So, you have to pretend that you're not using a variable in order to make coimpilation w.out ```dbus_enabled``` happy. That might be worth a comment; it's a pretty unusual situation.